.travis.yml

```diff
@@ -5,38 +5,159 @@ env:
     - secure: "R7uk+cduLPmjuUyzLCyaYkApE+WNIMrmvCy9cZxGy6N0HwOGnXt8dFgiZs6xB8PkePV2Q1sq0bO6QF8PT4GnnevjQtZb/BvARJtD2VfyI7y10t/hhKkK97Cx55GKKn+4M3hADSKfIphBHe4ZvUvA5gmswDW8AoQ4Lr53/6XK1msAxUa2XZHyg1D1F8jikWmQTcgyEw+8mF0BULWaKbzRhxiOsb2u9YTvBiMph/luf/sSOC18ZDN4MMSJshSq2UTH40Sz/8/ugjEMz7gFvMq+CDmJWYA68KiCuIB0Wx+UT77xzRH1LuIBAusVnkuBvKS83EnIHPjRkZ/5CAjyo5nTLPVwFBUyT1N5Pk8Mwbpe6rmtxqPbjTa0IoM+2WPxjYTbBWcr9HhtTp/sTRdKLLMYuj9aOT+D9LLg+dYepgfFoE84EkVu0eeGxSX+/P7mvaKaWEc/4HTS3AAStzWw5fM++x6zGmy6ru6s9v1wmFq89OYCB9Wrv+MpRjrurYw3C8n/WJR6Qyh7kIu1mjLcYM61axSULe8eET1vR2sAsbvgbwyxZIHHs6Ur1gS4s0cwqALAuuotwDuDAytK5X7MNYsZkJ6dHRw9gz107+ee4HeogW6TnNIip6htaUBPR6e9HGhJrMtK61LznGxyZrfaFqbj9FuI4mat0XJci8OZmFXJn68="
     - secure: "JV6xOo1YVqUUlsa/za7BGuYZj//y5vg7Dgcn6v0MJzF+ZxIM5maGh6iA3mhiCIUetHVwwlqjJg8ZaZ8vFcWRcx27+591xgTSJWrPhgnbqcNupDCGoexiqUUzfgOtkuDowmRp4wz2e5uUpox4Gm6gmJkEt1nUs+/gsk7fjDoChSTGgJ2eASiLgzY2Hg38iT5IiZMx//FWw0uQnYCtPR0vtGYdgT8aSMXnVSsFqvpUJBUnQj4R93eJbQiXUN6SCRpjvd536JOgd+ud+zm7QTvzmRKSgag31XDKNjD/1SK3zzhkfNL/1T7Kr3rCdluxa0RO2c3eLiN6rkjelVtFcRvj7zwJvZ2BJ5WLWILtvbvvCmNzI43v+8RandYiWm8CWxVwUt3Cw+xzG0DblO+TCT8P9DipljI5zOeEBh6Tc920zb0P45E5jnxu6oxS2sOV/K2oyFu5152JXI4u24xVxMeVd9cxOzd4Jpxn5pe/m7RPeEDTpHA28RBTjA98kysR3aUBsfF5I/IrxkT8e9XnMuAOGaVgaAzGRAjL481H8CMIkQT14hO3+jqPKcn1scgDOGxw+Ijujdix9BtVt5/pEokfXYd2KXBmgDAPTBqfgkabPDXoNv0iXDdkQVl0924s0Jq/Io1DbjIIS+zezrPOwryVBsPjRYfWsjwl9Nv1KjK+yeQ="
 
+
+# enable c++11/14 builds
 addons:
   apt:
-    sources: [ 'ubuntu-toolchain-r-test', 'llvm-toolchain-trusty-4.0' ]
-    packages: [ 'libstdc++-4.9-dev', 'clang-4.0' ]
+    sources: [ 'ubuntu-toolchain-r-test' ]
```

Comment:
Why removing the llvm toolchain source and clang?

Reply:
Because @danpat copied over the `.travis.yml` from node-cpp-skel which moves control of the compiler to `mason-js`, which installs a consistent version of clang++ via mason.

If anyone wants to add back jobs that use different compilers, feel free to do that. However node-cpp-skel only offers a template that uses clang++ via mason and g++ via 'ubuntu-toolchain-r-test' currently.

Reply:
There is a `# g++ build (default builds all use clang++)` build which adds `g++6` support, and shows how to override the Makefile default compiler paths.  Using system installed compilers is still fairly easy, just no longer the default.